src/System.Management.Automation/CoreCLR/CorePsAssemblyLoadContext.cs

```diff
@@ -1,19 +1,14 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
@iSazonov, your last commit had 38 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

Can load a native assembly.Can load native dll
```powershell
Expected $null or empty, but got D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.1\win7-x64\publish\mi.dll.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Basic\Assembly.LoadNative.Tests.ps1: line 64
64:         "${PSHOME}\${sourceDllName}" | Should -BeNullOrEmpty
```
Can load a native assembly.Can load native dll
```powershell
Expected $null or empty, but got C:\Users\VssAdministrator\AppData\Local\Temp\tmp7EA4.tmp\testDllNativeFolder\win-x64;C:\Users\VssAdministrator\AppData\Local\Temp\tmp7EA4.tmp\testDllNativeFolder\managed.dll.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Basic\Assembly.LoadNative.Tests.ps1: line 68
68:         (dir $root) -join ";" | Should -BeNullOrEmpty
```
Can load a native assembly.Can load native dll
```powershell
Expected $null or empty, but got C:\Users\VssAdministrator\AppData\Local\Temp\tmp7EA4.tmp\testDllNativeFolder\win-x64\nativedll.dll.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Basic\Assembly.LoadNative.Tests.ps1: line 72
72:         (dir $archFolder) -join ";" | Should -BeNullOrEmpty
```
Can load a native assembly.Can load native dll
```powershell
Exception calling "LoadNative" with "0" argument(s): "Unable to find an entry point named 'TestEntry' in DLL 'nativedll'."
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Basic\Assembly.LoadNative.Tests.ps1: line 80
80:         [TestNativeClass2]::LoadNative() | Should -Not -BeNullOrEmpty

```
Enter-PSHostProcess tests.By Process Id.Can enter using NamedPipeConnectionInfo
```powershell
Exception calling "Invoke" with "0" argument(s): "The runspace state is not valid for this operation."
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\Enter-PSHostProcess.Tests.ps1: line 136
136:                 $ps.AddScript('$pid').Invoke() | Should -Be $pwshId

```